P2.txt

```diff
@@ -0,0 +1,18 @@
+P2. Compute the GCD of 2 number:
+val a: Int

```

Comment:
The simple types are Integer, Character, String and Boolean (as defined in the Syntax.txt file), so the Int type (probably representing Integer) would be invalid